packages/flutter_rx_bloc/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  provider: ^6.0.0
+  provider: ^6.0.3
```

Comment:
When developing packages the constraints must be meaningful. If this package does not rely on the `provider` patches from 6.0.0-6.0.3 I would advise keeping the constraint to `provider: ^6.0.0`